index.html

```diff
@@ -19,9 +19,17 @@
         </a>
     </nav> 
 
+    <section class="interesting-notifications">
+      <p class="notif">
+        <span>New</span>
+        <strong>Crypto</strong>
+        -- New updates to the user dashboard
+        <a href="https://crypto.realdevsquad.com"> Check it out! ➡️ </a>
```

Comment:
What if we take the arrow emoji out of anchor tag?
The underline for anchor tag is going to the right of arrow emoji and it is looking a little weird. I initially thought it's a dot.
![image](https://user-images.githubusercontent.com/56217868/97825783-aff3a180-1ce5-11eb-8b62-392214430d24.png)

Reply:
This is how my emoji looks: 
![image](https://user-images.githubusercontent.com/1935403/97831452-ef3dd600-1c84-11eb-8db6-b638ef3c121e.png)

Reply:
I want the emoji to be clickable

Reply:
Even for emoji in your case, there's a little dot at the bottom right of the emoji. It's looking a bit weird to me. Plus, most of the users will come from Windows, it is looking a bit weird on Windows. Can we add a different id to it, still inside anchor tag but removing it's underline from CSS?

Just my opinion. Maybe we could compare the two - emoji not underlined and emoji underlined and see which one looks better?

Reply:
I changed the underline to none, and using a color transition now to denote that the link is clickable

Reply:
Cool. Thank you!